01-introduction.Rmd

```diff
@@ -108,18 +112,34 @@ leaflet() %>%
   addAwesomeMarkers(lng = c(-3, 23), lat = c(52, 53), popup = popup)
 ```
 
-It would be difficult to produce Figure \@ref(fig:interactive) with only 4 lines of code in another language, let alone embed the results in an interactive html page (the interactive version can be viewed at [robinlovelace.net/geocompr](http://robinlovelace.net/geocompr/intro.html)), illustrating R's flexibility. 
-
+It would be difficult to produce Figure \@ref(fig:interactive) with only four lines of code in another language, let alone embed the results in an interactive html page (the interactive version can be viewed at [robinlovelace.net/geocompr](http://robinlovelace.net/geocompr/intro.html)), illustrating R's flexibility. 
+The use of R code therefore enables teaching geocomputation with reference to reproducible examples such as that provided in \@ref(fig:interactive) rather than abstract concepts.
 <!-- Other reasons for using R for geocompuation include: -->
 
-R provides a stable language that behaves consistently across multiple computers, free of charge.
-The use of R code therefore enables teaching geocomputation with reference to reproducible examples such as that provided in \@ref(fig:interactive) rather than abstract concepts.
-But before cracking-on with the action, a few introductory remarks are needed to explain the approach taken here and provide context.
+### Python, Java, C++
```

Comment:
Yes that sounds good but I would call the section something more generic (and less scary for the newcomer!) like:

```
## Software for Geocomputation
```